actions/app.go

```diff
@@ -44,6 +49,22 @@ var (
 // `ServeFiles` is a CATCH-ALL route, so it should always be
 // placed last in the route declarations, as it will prevent routes
 // declared after it to never be called.
+
+// @title          Paphos API
+// @version        1.0
+// @description    Base backend API to serve Pygamillion UI
+// @termsOfService http://swagger.io/terms/
+
+// @contact.name   API Support
+// @contact.url    http://www.swagger.io/support
+// @contact.email  support@swagger.io
+
+// @license.name   Apache 2.0
+// @license.url    http://www.apache.org/licenses/LICENSE-2.0.html
+
+// @host                     tobedefined.com
```

Comment:
Minor fix suggestions:
- `Pygamillion` -> `Pygmalion`
- We don't have a TOS or support contacts, so we can probably just drop those fields instead
- Project license is AGPL 3.0, not Apache 2.0
- Host can be `localhost` since this will be served locally when developing

Reply:
No problem, I've pushed the changes (I kept the terms of service for swagger still)